pkg/inject/inject_test.go

```diff
@@ -23,7 +23,7 @@ func getConfig(fp func(Config) Config) Config {
 		IgnoredIPs:                  "169.254.169.254",
 		LogLevel:                    "debug",
 		Preview:                     false,
-		SidecarImage:                "840364872350.dkr.ecr.us-west-2.amazonaws.com/aws-appmesh-envoy:v1.20.0.1-prod",
+		SidecarImage:                "840364872350.dkr.ecr.us-west-2.amazonaws.com/aws-appmesh-envoy:v1.21.1.1-prod",
```

Comment:
would it be more appropriate to point it to the public ECR repo?
`public.ecr.aws/appmesh/aws-appmesh-envoy:v1.21.1.1-prod`